app/src/components/email.js

```diff
@@ -70,6 +84,18 @@ const email = {
     });
   },
 
+  /** Adds an email message to the queue
+   *  @param {object} message An email message object
+   *  @returns {object} A nodemailer result object
```

Comment:
what are the opts?  how would I know what I could set in here?

Reply:
Added the following to an upcoming commit. Thanks for the catch!
`@param {object} opts Bull job queue options to override default behavior. For more details on this object structure, refer to the JobOpts interface at https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queueadd.`